dev-client/src/screens/BottomTabsScreen.tsx

```diff
@@ -29,16 +30,36 @@ import {ProjectListScreen} from 'terraso-mobile-client/screens/ProjectListScreen
 import {UserSettingsScreen} from 'terraso-mobile-client/screens/UserSettingsScreen/UserSettingsScreen';
 
 export const BottomTabsScreen = memo(() => {
+  const [keyboardStatus, setKeyboardStatus] = useState(false);
```

Comment:
Let's extract this to a dedicated hook function - `useKeyboardStatus` or similar. I don't think it's appropriate for all this logic to live in this component.